packages/vincent-tools/vincent-tool-sdk/src/lib/handlers/evaluatePolicies.ts

```diff
@@ -72,7 +72,7 @@ export async function evaluatePolicies<
     const policy = vincentTool.policyMap.policyByPackageName[policyPackageName];
     try {
       const litActionResponse = await Lit.Actions.call({
-        ipfsId: policy.vincentPolicy.ipfsCid,
+        ipfsId: policy.ipfsCid,
```

Comment:
:+1: this will only be try if the `policy` is an `asBundledVincentPolicy()` result -- see my comment above about making sure that is what we export :+1: